bench-tps/src/bench.rs

```diff
@@ -750,8 +750,12 @@ mod tests {
     fn test_bench_tps() {
         let fullnode_config = FullnodeConfig::default();
         const NUM_NODES: usize = 1;
-        let cluster =
-            LocalCluster::new_with_config(&[999_990; NUM_NODES], 2_000_000, &fullnode_config, &[]);
+        let cluster = LocalCluster::new(&ClusterConfig {
+            node_stakes: vec![999_990; NUM_NODES],
+            cluster_lamports: 2_000_000,
+            fullnode_config,
+            ..ClusterConfig::default()
```

Comment:
I like it!

Reply:
@mvines pointed it out, something we saw @mark-solana do :)